db/inst/import-ForestGEO/import_species_list.R

```diff
@@ -37,7 +37,7 @@ spp <- read.csv(sppFile,stringsAsFactors=FALSE,strip.white = TRUE)
 spp <- spp[-duplicated(spp$Species),]
 
 ## open database connection & grab current species list
-dbparms.psql <- list(driver="PostgreSQL" , user = "bety", dbname = "bety", password='bety')
+dbparms.psql <- list(driver="PostgreSQL" , user = "bety", dbname = "bety", password='bety',host='128.197.230.32')
```

Comment:
@mdietze do you see this working for multiple dbs? Or would end users just all point to this one host for the forestGEO info?